plannotate/infernal.py

```diff
@@ -43,6 +43,8 @@ def parse_infernal(file_loc):
     infernal = infernal.apply(pd.to_numeric, errors='ignore', downcast = "integer")
     
     infernal['qseq'] = ""
+    to_swap = infernal['qend'] < infernal['qstart']
+    infernal.loc[to_swap, ['qstart', 'qend']] = infernal.loc[to_swap, ['qend', 'qstart']].values
```

Comment:
We have to be careful that this doesn't change the type to float64
```suggestion
    infernal.loc[to_swap, ['qstart', 'qend']] = infernal.loc[to_swap, ['qend', 'qstart']].values
    infernal[['qstart', 'qend']] = infernal[['qstart', 'qend']].apply(pd.to_numeric, downcast='integer')
```